packages/react-components/src/components/Picker/components/PickerTrigger.spec.tsx

```diff
@@ -71,7 +51,7 @@ describe('<PickerTrigger> component', () => {
       onClear,
     });
 
-    fireEvent.click(getByTestId(`${baseClass}__clear-icon`));
+    fireEvent.click(getByTestId(`picker-trigger__clear-icon`));
```

Comment:
IMO, we can switch to userEvent entirely, it is safer to use and more similar to normal user interactions.